deployments/Dockerfile.arm64

```diff
@@ -14,6 +14,7 @@ RUN  cd /usr/src/multus-cni && \
 FROM arm64v8/centos:7
 LABEL org.opencontainers.image.source https://github.com/k8snetworkplumbingwg/multus-cni
 COPY --from=build /usr/src/multus-cni /usr/src/multus-cni
+RUN yum update
```

Comment:
I have been having issues running yum update for arm based dockerfiles. I am yet to rootcause this, but this the error I get

```
standard_init_linux.go:211: exec user process caused "exec format error"
--
1191 | The command '/bin/sh -c yum update -y &&     yum clean all &&     rm -rf /var/cache/yum' returned a non-zero code: 1
```
Were you able to test it out ?